.github/workflows/check-formatting.yml

```diff
@@ -19,20 +19,21 @@ jobs:
       - name: Setup Go environment
         uses: actions/setup-go@v3
         with:
-          go-version: '^1.19.2'
+          go-version: 'stable'
 
       - name: Install formatters
         run: |
           wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
           sudo apt-get install software-properties-common
-          sudo add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-13 main'
-          sudo apt-get install clang-format-13
-          curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.42.1/ktlint && chmod a+x ktlint && sudo mv ktlint /usr/bin/ktlint
+          sudo add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-15 main'
```

Comment:
Unless updating to 15 gives us much, could we stick to 14? LLVM 15 isn't available in the official Ubuntu 22.04 sources, which would make it harder to run `./format.sh` locally.

Reply:
No major gains their that I'm aware of. Changed to LLVM 14